src/misc/register-service-worker.js

```diff
@@ -30,3 +30,12 @@ if (process.env.NODE_ENV === 'production') {
     }
   })
 }
+
+let refreshing = false
```

Comment:
I do not understand this part. When is `refreshing` set to true ?

Reply:
When a new Service Worker take the control ;)